### ISSUE TITLE
[Accessibility] Keyboard Handling does not work properly

### ISSUE THREAD

**Issue by ihardyslide** [Vuepic/vue-datepicker#932]
Hi,

sorry for opening up another bug, but I did not get an answer on my [closed one](https://github.com/Vuepic/vue-datepicker/issues/822#issuecomment-2160244697), where I wrote an answer.
Can you just give me a statement if you will not look into this anymore or if you just have not seen my answer?

Thanks & BR

**Comment by Jasenkoo** [comment]
Yes, I saw it, will be handled

**Comment by ihardyslide** [comment]
Hi,

I am feeling bad already for coming back to this issue again, but unfortunately the latest version still has [both issues I mentioned](https://github.com/Vuepic/vue-datepicker/issues/822#issuecomment-2160244697), I can reproduce them in your playground with version 9.0.1
If I can help somehow with fixing those issues, I would be up for it.

Thanks & BR
